src/components/search-form/SearchForm.scss

```diff
@@ -17,11 +17,6 @@
         padding: 0;
         width: 30px;
 
-        &:hover,
```

Comment:
This is also used in ContentExplorer and MetadataSidebar. Can you check those quickly to make sure they don't have any glaring issues after this change?

Reply:
Just took a look at both and no glaring issues.  Only difference is the outline shows up now when you tab to it.